README.md

```diff
@@ -113,6 +114,31 @@ svc/example-client     ClusterIP   10.31.243.173   <none>        2181/TCP
 svc/example-headless   ClusterIP   None            <none>        2888/TCP,3888/TCP   2m
 ```
 
+### Upgrade a Zookeeper cluster
+
+To initiate an upgrade process, a user has to update the `spec.image.tag` field of the `ZookeeperCluster` custom resource. This can be done in three different ways using the `kubectl` command.
+1. `kubectl edit zk <name>`, modify the `tag` value in the YAML resource, save, and exit.
+2. If you have the custom resource defined in a local YAML file, e.g. `zk.yaml`, you can modify the `tag` value, and reapply the resource with `kubectl apply -f zk.yaml`.
+3. `kubectl patch zk <name> --type='json' -p='[{"op": "replace", "path": "/spec/image/tag", "value": "X.Y.Z"}]'`.
+
+After the `tag` field is updated, the StatefulSet will detect the version change and it will trigger the upgrade process.
+
+However, the value of this `tag` field should not be modified field while an upgrade is already in progress.
```

Comment:
Please add this line is as a separate "Note:"
at the end of the Upgrade block:
`However, the value of this `tag` field should not be modified field while an upgrade is already in progress.`

Reply:
updated